crates/node/src/interactive_cli/context.rs

```diff
@@ -47,6 +47,12 @@ enum Commands {
     Create {
         /// The application to create the context with
         application: Alias<ApplicationId>,
+         /// Name alias for the new context
+         #[clap(long)]
+         name: Option<Alias<ContextId>>,
+         /// Identity alias for the creator
+         #[clap(long = "as")]
+         as_alias: Option<Alias<PublicKey>>,
```

Comment:
```suggestion
         #[clap(long = "as")]
         author: Option<Alias<PublicKey>>,
```

`author` / `creator`